nextcloud-startup-script.sh

```diff
@@ -360,34 +337,6 @@ sed -i "s|;date.timezone.*|date.timezone = $(cat /etc/timezone)|g" "$PHP_INI"
 occ_command config:system:set logtimezone --value="$(cat /etc/timezone)"
 clear
 
-# Check where the best mirrors are and update
-msg_box "To make downloads as fast as possible when updating you should have mirrors that are as close to you as possible.
-This VM comes with mirrors based on servers in that where used when the VM was released and packaged.
-
-If you are located outside of Europe, we recomend you to change the mirrors so that downloads are faster."
-print_text_in_color "$ICyan" "Checking current mirror..."
-print_text_in_color "$ICyan" "Your current server repository is: $REPO"
-
-if [[ "no" == $(ask_yes_or_no "Do you want to try to find a better mirror?") ]]
-then
-    print_text_in_color "$ICyan" "Keeping $REPO as mirror..."
-    sleep 1
-else
-    print_text_in_color "$ICyan" "Locating the best mirrors..."
-    apt update -q4 & spinner_loading
-    apt install python-pip -y
-    pip install \
-        --upgrade pip \
-        apt-select
-    check_command apt-select -m up-to-date -t 5 -c -C "$(localectl status | grep "Layout" | awk '{print $3}')"
-    sudo cp /etc/apt/sources.list /etc/apt/sources.list.backup && \
-    if [ -f sources.list ]
-    then
-        sudo mv sources.list /etc/apt/
-    fi
-fi
-clear
-
```

Comment:
why did you remove that?

Reply:
`python-pip` doesn't exist in 20.04 yet, so I moved it to a script instead so that we easily can add it back later.

The script itself isn't perfect either: https://github.com/nextcloud/vm/issues/1155

If you have time, maybe you could have a look?

Reply:
This is now fixed.